WcaOnRails/spec/lib/relations_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require 'relations'
+
+RSpec.describe "Relations" do
+  before do
+    allow(Relations).to receive(:linkings).and_return(
```

Comment:
Actually this should be as effective as populating the table and this looks clean IMO. Also note that this would require the effort of adding a migration and ensuring that there's really some table just for the test purposes, as it's going to be generated with the script anyway.